src/BookNavigator/volumes/volumes-provider.js

```diff
@@ -0,0 +1,73 @@
+import { html } from 'lit-element';
+import sortAscendingIcon from '../assets/icon_sort_ascending.js';
+import sortDescendingIcon from '../assets/icon_sort_descending.js';
+
+import './volumes.js';
+
+export default class VolumesProvider {
+
+  constructor(optionChange, baseHost, bookreader) {
+    this.optionChange = optionChange;
+    this.component = document.createElement('viewable-files');
+
+    const files = bookreader.options.multipleBooksList?.by_subprefix;
+    this.viewableFiles = Object.keys(files).map(item => files[item]);
+    this.volumeCount = Object.keys(files).length;
+    this.isFirstLoad = true;
+    this.isSortAscending = false;
+
+    this.component.subPrefix = bookreader.options === undefined ? '' : bookreader.options.subPrefix;
```

Comment:
q: why would bookreader.options be undefined? 
given the way this menu instantiates & behaves, i want to argue that `bookreader.options.subPrefix` _is_ a required value, an empty string as default is just to protect from type errors..

let's lean on the falsy and do this:

```suggestion
    this.component.subPrefix = bookreader.options.subPrefix || '';
```